### ISSUE TITLE
Facebook login form automatically closes when FB SDK is not autoinitialized on startup

### ISSUE THREAD

**Issue by TatyanaLeschenok** [facebook/facebook-ios-sdk#1447]
## Checklist

- [x] I've updated to the latest released version of the SDK
- [x] I've searched for existing [GitHub issues](https://github.com/facebook/facebook-ios-sdk/issues)
- [x] I've looked for existing answers on [Stack Overflow](https://facebook.stackoverflow.com), the [Facebook Developer Community Forum](https://developers.facebook.com/community/) and the [Facebook Developers Group](https://www.facebook.com/groups/fbdevelopers)
- [x] I've read the [Code of Conduct](CODE_OF_CONDUCT.md)
- [x] This issue is not security related and can safely be disclosed publicly on GitHub

## Environment

- Xcode Version: `11.5`
- Installation Platform & Version: `[Manual]` version `7.1.1`

## Expected Results

When SDK autoinitialization disabled and a user needs to tap on "Continue with Facebook" button once to log in

## Actual Results

When SDK autoinitialization disabled and a user taps on "Continue with Facebook" button Facebook log in form opens and closes, so a user has to tap the button again to show Log in form

## Steps to Reproduce

1. Disable SDK autoinitialization by adding `FacebookAutoInitEnabled` plist entry (as mentioned in [Getting started guide](https://developers.facebook.com/docs/app-events/getting-started-app-events-ios))
2. Manually init sdk (`ApplicationDelegate.initializeSDK(nil)`) before requesting access token with a set of permissions using this api [FBSDKLoginManager](https://github.com/facebook/facebook-ios-sdk/blob/master/FBSDKLoginKit/FBSDKLoginKit/FBSDKLoginManager.h#L142)
3. Tap "Continue with Facebook" button
4. Tap "Continue" on the dialog asking to allow your app to access facebook.com
5. Facebook Web Login form presented to the user for a moment and automatically closed and the app receives `LoginResult` .cancelled.

## Code Samples & Details

Please checkout [this branch](https://github.com/TatyanaLeschenok/facebook-ios-sdk/tree/fb-login-samle-app-disable-autoinit) that applies described steps for the FacebookLoginSample app **Login Manager 
Example** part to illustrate the issue.

1. Bump SDK to use the latest 7.1.1 version
2. Disable auto init of FB SDK
```
<key>FacebookAutoInitEnabled</key>
<false/>
```

3. Manually initialize FB SDK before log in attempt

```swift
@IBAction func toggleLoginState() {
        ApplicationDelegate.initializeSDK(nil)
        let loginManager = LoginManager()

        guard !isLoggedIn else {
            loginManager.logOut()
            return updateLoginButton()
        }

        loginManager.logIn(
            permissions: [.publicProfile, .email],
            viewController: self
        ) { [unowned self] result in

            switch result {
            case .failed(let error):
                self.presentAlert(for: error)

            case .cancelled:
                self.presentAlert(title: "Cancelled", message: "Login attempt was cancelled")

            case .success:
                self.updateLoginButton()
                self.showLoginDetails()
            }
        }
    }
```

**Comment by TatyanaLeschenok** [comment]
Same issue was reported on developer.facebook.com https://developers.facebook.com/support/bugs/758221278262132/

**Comment by sanvean** [comment]
Having the exact same issue using 7.1.1 via SPM.

**Comment by sanvean** [comment]
@TatyanaLeschenok btw until this is fixed we found adding a small delay post-initialisation of the FB SDK (up to ~0.75 secs to cover older iOS 12 devices) before initiating the login action seems to eradicate the issue with good network conditions.

**Comment by sanvean** [comment]
@TatyanaLeschenok fyi this looks like it's fixed in v8.0.0

**Comment by joesus** [comment]
Should be fixed in 8.0.0. Please reach out if this is not the case.

**Comment by firatagdas** [comment]
This issue still on going but I don't have Facebook Auto init has disabled explicitly for iOS 14 and there is no error log.
